### PR TITLE
Sac/ABR/Cre changes

### PR DESCRIPTION
**Pull request by Sacristan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,48 @@
 # Mailigen
-
-[![Gem Version](https://badge.fury.io/rb/mailigen.png)](http://badge.fury.io/rb/mailigen)
```
Comment (artursbraucs) [inline README.md:3]: why badges were removed?

File: README.md [hunk 2]
```diff
@@ -1,63 +1,48 @@
...
-
-And then execute:
-
-    $ bundle
-
-Or install it yourself as:
-
-    $ gem install mailigen
+```rb
+gem 'mailigen', git: 'https://github.com/CreativeGS/mailigen', ref: "efd8cf899d938dddb50f649826071978453a0a81"
```
Comment (artursbraucs) [inline README.md:7]: Please, bring back previous installation guide.

File: mailigen.gemspec [hunk 3]
```diff
@@ -19,10 +19,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.required_ruby_version = '~> 2.0'
-  
-  spec.add_dependency "activesupport"
+
+  spec.add_dependency "activesupport", ">= 3.0"
+  spec.add_dependency "retryable", ">= 2.0.4" # retry flaky remote requests
```
Comment (artursbraucs) [inline mailigen.gemspec:24]: I don't see any place where you using it.

File: mailigen.gemspec [hunk 4]
```diff
@@ -19,10 +19,13 @@ Gem::Specification.new do |spec|
...
 
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec"
+  spec.add_development_dependency "rspec", "~> 3.5.0"
+  spec.add_development_dependency "pry", "~> 0.10.4"
```
Comment (artursbraucs) [inline mailigen.gemspec:29]: I think it is unnecessary dep for library. Is there any reason why it should be there?

**Review by artursbraucs — CHANGES_REQUESTED** [review]

**Comment by Sacristan** [comment]
Nvm! This is a Creative.gs implementation!

Please close pull request.

**Pull request closed without merging**